product/roundhouse.databases.sqlserver2000/SqlServerDatabase.cs

```diff
@@ -93,15 +93,19 @@ public override void run_database_specific_tasks()
 
         public override string create_database_script()
         {
+            //Using global variable @@VERSION to get the information about current MS SQL Server instance.
             return string.Format(
                 @"
                         DECLARE @Created bit
                         SET @Created = 0
-                        IF NOT EXISTS(SELECT * FROM sys.databases WHERE [name] = '{0}') 
-                         BEGIN 
-                            CREATE DATABASE [{0}] 
-                            SET @Created = 1
-                         END
+                        IF(PATINDEX('%Microsoft SQL Server 2000%',@@VERSION) > 0)

```

Comment:
In my view, no need to check the version of SQL Server. Anyway, this code is in the specialized class SQL2000 and it will never call for other server types.